src/index.html

```diff
@@ -7,8 +7,35 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo-container">
+        <a href="#home"><img src="images/logo.png" alt="logo"></a>
+      </div>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__list-item"><a href="#apple" class="header__nav-link is-active">Apple</a></li>
```

Comment:
[Сhecklist #4](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md). Your lines should be below 80 symbols, check the entire file
```suggestion
          <li class="header__list-item">
            <a href="#apple" class="header__nav-link is-active">Apple</a>
          </li>
```